### PR TITLE
docs: updates catalog-backend-module-aws with setup steps

### PR DESCRIPTION
**Pull request by wanisfahmyDE**
## Hey, I just made a Pull Request!

This PR updates catalog-backend-module-aws with setup steps.
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/catalog-backend-module-aws/README.md [hunk 1]
```diff
@@ -4,7 +4,56 @@ This is an extension module to the plugin-catalog-backend plugin, providing an
 `AwsOrganizationCloudAccountProcessor` that can be used to ingest cloud accounts
 as `Resource` kind entities.
 
-## Getting started
+The plugin also supports discovering catalog entities located in an S3 Bucket. See the [Backstage documentation](https://backstage.io/docs/integrations/aws-s3/discovery) for more details.
 
-See [Backstage documentation](https://backstage.io/docs/integrations/aws-s3/discovery) for details on how to install
-and configure the plugin.
+## For Ingesting AWS accounts
```
Comment (Rugvip) [inline plugins/catalog-backend-module-aws/README.md:9]: This is more or less the same as what's available at https://backstage.io/docs/integrations/aws-s3/discovery/ right? We've discussed this balance a bit back and forth, wanting to have self-contained READMEs but also avoid duplication. So out of curiosity, how come you want to add these instructions to the README as well?

Comment (pjungermann) [reply plugins/catalog-backend-module-aws/README.md:9]: At other providers like the one for GitHub, we have additional pages for other providers like those for org data: https://backstage.io/docs/integrations/github/org

This one is not about org data and not a provider (maybe, it was never migrated to entity providers).

I wonder whether or whether not we want to have it more at one place for the plugin.

File: plugins/catalog-backend-module-aws/README.md [hunk 2]
```diff
@@ -4,7 +4,56 @@ This is an extension module to the plugin-catalog-backend plugin, providing an
...
+```bash
+# From your Backstage root directory
+yarn add --cwd packages/backend @backstage/plugin-catalog-backend-module-aws
+```
+
+2. Import the plugin to `catalog.ts` and add the `AwsOrganizationCloudAccountProcessor`
+
+```TypeScript
+// packages/backend/src/plugins/catalog.ts
++ import {AwsOrganizationCloudAccountProcessor} from '@backstage/plugin-catalog-backend-module-aws';
```
Comment (pjungermann) [inline plugins/catalog-backend-module-aws/README.md:22]: The formatting of the code seems not to follow the prettier formatting used for the project.

File: plugins/catalog-backend-module-aws/README.md (hunk 1 above)
Comment (pjungermann) [reply plugins/catalog-backend-module-aws/README.md:9]: PS: there are more processors (we should omit the deprecated ones replaced by a provider)

Comment (wanisfahmyDE) [reply plugins/catalog-backend-module-aws/README.md:9]: sure, should I move the changes somewhere here?https://github.com/backstage/backstage/tree/master/docs/integrations/github

File: plugins/catalog-backend-module-aws/README.md (hunk 2 above)
Comment (wanisfahmyDE) [reply plugins/catalog-backend-module-aws/README.md:22]: hmm I ran `npx prettier --write README.md` locally and pushed these changes, and the Verify workflow logs are happy in the `prettier` [step](https://github.com/backstage/backstage/actions/runs/5918320281/job/16047533982?pr=19467):

```bash
Run yarn prettier:check
Checking formatting...
All matched files use Prettier code style!
```
Can you maybe point me out to what's missing?

File: plugins/catalog-backend-module-aws/README.md (hunk 1 above)
Comment (Rugvip) [reply plugins/catalog-backend-module-aws/README.md:9]: Yep let's move it over to the docs. I think a new `aws` dir in https://github.com/backstage/backstage/blob/master/docs/integrations/ is the way to go tbh. Then we can think about whether we want to move in `aws-s3` into that in the future.

**Review by Rugvip — CHANGES_REQUESTED** [review]

**Pull request closed without merging**